CONTRIBUTING.md

```diff
@@ -16,21 +20,37 @@ To submit a bug report or a feature request, please open an [issue](https://gith
 
 Follow these steps to start contributing code:
 
+1. Setup your environment i.e:
+```bash
+conda create -n trlx python=3.8 -c pytorch
+```
 1. Create your own [fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo#forking-a-repository) of the repository and clone it to your local machine.
     ```bash
     git clone https://github.com/<YOUR-USERNAME>/trlx.git
     cd trlx
     git remote add upstream https://github.com/CarperAI/trlx.git
     ```
-2. Create a new branch for your changes and give it a concise name that reflects your contribution.
+1. Create a new branch for your changes and give it a concise name that reflects your contribution.
```

Comment:
Seems like the numbering got messed up here

Reply:
Markdown handles it automatically so you don't have to change it. 
```
1. is one
1. is one
1. Is one
```
1. is one
1. is one
1. Is one